SwiftUI-Xcode-Template.xcodeproj/project.pbxproj

```diff
@@ -412,122 +407,150 @@
 			};
 			name = Release;
 		};
-		3F858F6726C34B4D0002B27F /* Debug */ = {
+		3F6F0A8226C34BE7008FA0A2 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				ASSETCATALOG_COMPILER_GLOBAL_ACCENT_COLOR_NAME = AccentColor;
 				CODE_SIGN_STYLE = Automatic;
+				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"SwiftUI-Xcode-Template/Preview Content\"";
 				ENABLE_PREVIEWS = YES;
-				INFOPLIST_FILE = "SwiftUI-Xcode-Template/Info.plist";
+				GENERATE_INFOPLIST_FILE = YES;
+				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
+				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
+				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPhone = "UIInterfaceOrientationPortrait UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight";
```

Comment:
These settings and the `CURRENT_PROJECT_VERSION` and `MARKETING_VERSION` ones a few lines above and below represent the new default format that replaces the `Info.plist`.

I'd say if you move these settings into an `.xcconfig`, managing them would become easier than if they were in the `.plist` XML format. 🤷‍♂️